services/spar/src/Spar/Run.hs

```diff
@@ -96,7 +97,7 @@ runServer sparCtxOpts = do
   let settings = Warp.defaultSettings
         & Warp.setHost (fromString $ sparCtxOpts ^. to saml . SAML.cfgSPHost)
         . Warp.setPort (sparCtxOpts ^. to saml . SAML.cfgSPPort)
-  sparCtxHttpManager <- newManager defaultManagerSettings
+  sparCtxHttpManager <- newManager tlsManagerSettings
```

Comment:
working on this here: https://github.com/wireapp/wire-server/compare/sso-integration-tests-2...sso-integration-tests-3?expand=1

the mock server tls works, and now spar refuses to create the idp because it cannot validate the tls certificate in the https handshake.  i need more time to figure out how to do this in http-client-tls or elsewhere, (brig doesn't appear to have this problem.)

if you give me green light now, we can have this PR deployed by the time QA runs the regression tests again tonight.

all the other comments were good, i think i fixed everything.  thanks!